autoextract_spiders/middlewares.py

```diff
@@ -0,0 +1,25 @@
+from fake_useragent import UserAgent
+from scrapy.downloadermiddlewares.useragent import UserAgentMiddleware
+
+
+class RandomUserAgentMiddleware(UserAgentMiddleware):
+
+    @classmethod
+    def from_crawler(cls, crawler):
+        user_agent, random_type = crawler.settings['USER_AGENT'], None
+        if crawler.settings.get('FAKEUSERAGENT_ENABLED'):
```

Comment:
it is better to use getbool here, this way command line options would be handled properly